frontend/talentsearch/src/js/components/browse/PoolCard.tsx

```diff
@@ -0,0 +1,178 @@
+import { Link } from "@common/components";
+import { getPoolStream } from "@common/constants/localizedConstants";
+import React from "react";
+import { useIntl } from "react-intl";
+import Chip, { Chips } from "@common/components/Chip";
+import {
+  getLocale,
+  getLocalizedName,
+  localizeSalaryRange,
+} from "@common/helpers/localize";
+import { formattedDateMonthDayYear } from "@common/helpers/dateUtils";
+import { PoolAdvertisement } from "../../api/generated";
+import { useDirectIntakeRoutes } from "../../directIntakeRoutes";
+
+export interface CardProps {
+  pool: PoolAdvertisement;
+}
+
+const PoolCard = ({ pool }: CardProps & React.HTMLProps<HTMLDivElement>) => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+  const paths = useDirectIntakeRoutes();
+
+  const poolClassificationString = (): string | null => {
+    if (pool.classifications && pool.classifications[0]) {
+      const classification = pool.classifications[0];
+      return `${classification.group}-0${classification.level}`;
+    }
+    return null;
+  };
```

Comment:
Not a *huge* deal, but I think we could move this outside of the component and make it a pure function so we aren't executing it twice later on.

```tsx
const poolClassificationString = (
  classifications: Maybe<Maybe<Classification>[]>,
): string | null => {
  if (!classifications || !classifications[0]) return null;

  const classification = classifications[0];

  return `${classification.group}-0${classification.level}`;
};

...

const classification = poolClassificationString(pool.classifications);

...

{classification || intl.formatMessage(...)}
```

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/pull/4233/commits/1ce95b75f58cdecd76253fc5d31dd4c84f5a5beb